### PR TITLE
Changelog for 2.3.4

### PR DESCRIPTION
**Pull request by daschuer**

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
https://github.com/mixxxdj/mixxx/pull/4729 is nearly ready too

**Comment by JoergAtGithub** [comment]
Could we highlight the new mapping (Traktor Kontrol S2 Mk1) somehow. Maybe by moving it on top of the controller changes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +34,32 @@
...
 * Fixed crash with FFmpeg decoder [#11044](https://github.com/mixxxdj/mixxx/pull/11044)
 * Fixed issue with finding moved library tracks. [#11051](https://github.com/mixxxdj/mixxx/pull/11051)
+* Preserve and synchronize ID3v1 tags (TagLib v1.12) [#11163](https://github.com/mixxxdj/mixxx/pull/11163) [#11123](https://github.com/mixxxdj/mixxx/issues/11123)
+* Replay Gain Preferences: Fix the "adjust by" text in case of negative adjustments [#11176](https://github.com/mixxxdj/mixxx/pull/11176)
+* macOs: Install Qt translation [#11134](https://github.com/mixxxdj/mixxx/pull/11134) [#11110](https://github.com/mixxxdj/mixxx/issues/11110)
+* macOs: Fix assuming wrong system language [#11218](https://github.com/mixxxdj/mixxx/pull/11218) [#11195](https://github.com/mixxxdj/mixxx/issues/11195)
+* Fix resetting track colors on metadata reimport (Serato metadata): [#11217](https://github.com/mixxxdj/mixxx/pull/11217) [#11213](https://github.com/mixxxdj/mixxx/issues/11213)
+* Preferences: Fix incomplete version check in 2.3 during upbgade [#11229](https://github.com/mixxxdj/mixxx/pull/11229) [#9709](https://github.com/mixxxdj/mixxx/issues/9709)
```
Comment (ronso0) [inline CHANGELOG.md:42]:
```suggestion
* Preferences: Fix incomplete version check in 2.3 during upgrade [#11229](https://github.com/mixxxdj/mixxx/pull/11229) [#9709](https://github.com/mixxxdj/mixxx/issues/9709)
```

**Comment by daschuer** [comment]
> Could we highlight the new mapping (Traktor Kontrol S2 Mk1) somehow. Maybe by moving it on top of the controller changes?

I have added a new section for new mappings

> https://github.com/mixxxdj/mixxx/pull/4729 is nearly ready too

When do you think it is ready? 
There is always one last PR, blocking a release. I suggest to merge this PR independently. 
If we manage to finish it in time we can add the change log entry independently.

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +23,7 @@
   [#10979](https://github.com/mixxxdj/mixxx/pull/10979)
   [#10973](https://github.com/mixxxdj/mixxx/pull/10973)
   [#10983](https://github.com/mixxxdj/mixxx/pull/10983)
+  [#11288](https://github.com/mixxxdj/mixxx/pull/11288)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:26]: Typo: performaces -> performance

File: CHANGELOG.md [hunk 3]
```diff
@@ -43,6 +35,40 @@
 * Auto DJ: Added a warning ina message box when it is started without decks with left and a right crossfader orientation [#11018](https://github.com/mixxxdj/mixxx/pull/11018)
```
Comment (JoergAtGithub) [inline CHANGELOG.md:35]:
```suggestion
* Auto DJ: Added a warning in a message box when it is started without decks with left and a right crossfader orientation [#11018](https://github.com/mixxxdj/mixxx/pull/11018)
```

**Comment by daschuer** [comment]
Done

**Comment by JoergAtGithub** [comment]
LGTM! Thank you!

**Pull request merged**